hackernews/program.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import time
+import urllib.parse
+
+from autokitteh.slack import slack_client
+
+
+slack = slack_client("slack_connection")
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL")
+
+
+def on_command(event):
+    topic = event.data.text  # Extract the topic from the Slack command
+    existing_articles = set()
+    fetch_articles(topic, existing_articles)
+
+    while True:
+        all_articles = set(existing_articles)
```

Comment:
I think the naming is confusing here.
Maybe `latest_articles` or `recent_articles` or `current_articles` instead of `existing_articles`?